### PR TITLE
I205 email incidents and vet reports

### LINKED ISSUE #205: Email incidents and vet reports
**Is your feature request related to a problem? Please describe.**
Currently, Incident and Vet reports are not emailed to POOPS.

**Describe the solution you'd like**
Incident and Vet reports should be emailed to POOPS.

For testing purposes, you could use your own email.

### PR DESCRIPTION
**Pull request by pavo-etc**
## Change Summary
Adds email support to incident reporting and vet reporting.  This required a number of Firebase changes, with emails being sent from the poops.cfc.test@gmail.com account. (Ask Zach for user account details).  The emails will be send to the INCIDENT_EMAIL address in the `.env` file.  See `.env.example`.

### Change Form
N/A

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [x] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related Issue

- Resolve #205

### CODE REVIEW

**Comment by pavo-etc** [comment]
Still need to write documentation for this

File: src/pages/api/sendEmail.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+interface Message {
+  message: {
+    subject: string
+    text: string
+    html: string
+  }
+}
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
```
Comment (NicholasChoong) [inline src/pages/api/sendEmail.ts:13]: This should be a firebase functions. I can create a new issue to convert this function into a firebase function

**Review by NicholasChoong — COMMENTED** [review]
looks god

**Review by NicholasChoong — APPROVED** [review]

**Review by NicholasChoong — APPROVED** [review]
looks good

**Review by NicholasChoong — APPROVED** [review]

File: src/pages/api/sendEmail.ts (hunk 1 above)
Comment (pavo-etc) [reply src/pages/api/sendEmail.ts:13]: Yeah I think do this as a separate issue

**Pull request merged**